states/pacman.mod.py

```diff
@@ -0,0 +1,308 @@
+import threading
+import socket
+import json
+from PIL import Image, ImageDraw, ImageFont
+from time import sleep
+from random import choice
+
+from states.base import BaseState
+
+
+class Game:
+    def __init__(self):
+        self.image = Image.new("RGB", (96, 32), "black")
+        self.draw = ImageDraw.Draw(self.image)
+        self.food = 0
+        f = open("./static/pacman_assets/map.json")
```

Comment:
```suggestion
        with open("./static/pacman_assets/map.json") as f:
```
en dan kan f.close() weg.